docs/administrators/upgrading_the_orchestrator.rst

```diff
@@ -0,0 +1,100 @@
+.. _upgrading_the_orchestrator:
+
+
+Upgrading the orchestrator
+-------------------------
+
+Migrating from one running orchestrator to another running orchestrator
+#######################################################################
+
+This document describes how to upgrade to a new version of the orchestrator by setting
+up a new orchestrator next to the existing orchestrator and migrating all the state from
+the existing to the new orchestrator. This procedure should be followed when an in-place
+update of the orchestrator is not possible e.g. when the operating system needs to be
+updated alongside the orchestrator.
+
+Terminology
++++++++++++
+
+The procedure below describes how to migrate from one running orchestrator
+denoted as the 'old orchestrator' to another one denoted as the 'new orchestrator'.
+
+Procedure
++++++++++
+
+
+.. note::
+    **Pre-requisite**
+
+    - Before upgrading the orchestrator to a new major version, an update should be done first.
+    - Upgrades should be done one major version at a time. So if you want to upgrade from major
+      version :code:`X` to major version :code:`X+2`, you should do an upgrade from :code:`X` to :code:`X+1` and then from :code:`X+1` to :code:`X+2`.
+
+
+_________
+
+1. **[New Orchestrator]**: Make sure the desired version of the orchestrator is installed, by following the
+installation instructions (see :ref:`install-server`) and set up a project manually, validating that the setup
+of the orchestrator works (config, credentials, access to packages, etc.).
+
+_________
+
+
+2. **[Old Orchestrator]** Halt all environments (by pressing the ``STOP`` button in the web-console for each environment).
+3. **[Old Orchestrator]** Stop the server:
+
+.. code-block:: bash
+
+    sudo systemctl disable --now inmanta-server.service
+
+4. **[Old Orchestrator]** Make a dump of the server database using ``pgdump``.
+
+
+.. code-block:: bash
+
+    pg_dump -U <user> -W -h <host> <db_name> > <db_dump_file>
+
+_________
+
+
+
+5. **[New Orchestrator]** Make sure the server is stopped:
+
+.. code-block:: bash
+
+    sudo systemctl stop inmanta-server.service
+
+6. **[New Orchestrator]** Load the dump of the server database using ``psql``.
```

Comment:
You may have to drop the DB first, as it already has the DB initialized on the wrong version .

Did you test this procedure?

Reply:
Not yet, I will test it out now that the procedure is fleshed out